### PR TITLE
DOC-579 | ArangoGraph credits & usage

### PR DESCRIPTION
**Pull request by nerpaula**
### Description

Added Credits & Usage feature description and FAQs. 

Preview page: https://deploy-preview-300--docs-hugo.netlify.app/stable/arangograph/organizations/credits-and-usage/

### CODE REVIEW

File: site/content/3.10/arangograph/organizations/credits-and-usage.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+---
+title: Credits & Usage in ArangoGraph
```
Comment (Simran-B) [inline site/content/3.10/arangograph/organizations/credits-and-usage.md:2]: Credits and their usage in ArangoDB?

File: site/content/3.10/arangograph/organizations/credits-and-usage.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+menuTitle: Credits & Usage
+weight: 15
+description: >-
+   Credits give you access to a flexible prepaid model, so you can allocate them
+   across multiple deployments as needed
```
Comment (Simran-B) [inline site/content/3.10/arangograph/organizations/credits-and-usage.md:7]:
```suggestion
  Credits give you access to a flexible prepaid model, so you can allocate them
  across multiple deployments as needed
```

File: site/content/3.10/arangograph/organizations/credits-and-usage.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+2. Click **Credits & Usage** in the **Organization** section.
+3. In the **Credits & Usage** page, you can:
+   - See the remaining credit balance.
+   - Track your total credit balance.
+   - See a projection of when you will run out of credits, based on the last 30 days of usage. 
+   - Get a detailed consumption report in PDF format that shows:
+      - The number of credits you had at the start of the month.
+      - The number of credits consumed in the month.
+      - The number of credits remaining.
+      - The number of credits consumed for each deployment.
```
Comment (Simran-B) [inline site/content/3.10/arangograph/organizations/credits-and-usage.md:61]:
```suggestion
     - The number of credits you had at the start of the month.
     - The number of credits consumed in the month.
     - The number of credits remaining.
     - The number of credits consumed for each deployment.
```

File: site/content/3.10/arangograph/organizations/credits-and-usage.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+All details about credits, including how many credits have been purchased,
+how many remain, and how they are being consumed are available in the
+**Credits & Usage** page within the ArangoGraph web interface.
+
+### I have a large sharded deployment, how do I know how many credits it will consume?
+
+Each node size has a per-hour credit price for a given machine. You can calculate
+the price of a sharded deployment by multiplying the machine price by the number
+of machines in the deployment. You can download a detail consumption report from
```
[the remark this answers is not in the document]
Comment (Simran-B) [reply site/content/3.10/arangograph/organizations/credits-and-usage.md:131]: The Pricing section isn't covered in the docs

[the remark this answers is not in the document]
Comment (mattcarabine) [reply site/content/3.10/arangograph/organizations/credits-and-usage.md:131]: @nerpaula looks like this wasn't addressed

**Comment by Simran-B** [comment]
@mattcarabine  I don't have access to the Avocado organization in dev to retake the screenshot and the Support Organization I am in shows something strange for the credits projection:

image

The consumption is between 200 and 400 credits and the estimate for remaining credits goes from >33k down to 0 within a month?

**Comment by mattcarabine** [comment]
@Simran-B that's expected, as the credits expire by that date (so you will have 0).

**Comment by nerpaula** [comment]
@mattcarabine the screenshot has been retaken, and all other suggested changes are applied (thanks @Simran-B).
Please re-review, then we are good to merge.

**Review by mattcarabine — APPROVED** [review]

**Pull request merged**